portfolio/src/main/webapp/hiking.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>ROB MARCUS x HIKING</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>
+      <a href="index.html" style="text-decoration: none; color: black">
+        ROB MARCUS
+      </a>
+      <div> x HIKING</div>
+    </h1>
+    <div id = "content">
+    <p>Material will go here.</p>
+    </div>
+
+    <div id = "content">
```

Comment:
ID is unique.  Only one element should have a particular ID.  Also in other places.

If you have a group of similar elements and are trying to apply the same style for all of them, use classes.

For some perspective, classes are *far* more common in production HTML code than ids.  We'll give one element multiple classes and use those classes as descriptors for a particular set of styles to apply to it or a particular grouping of elements that it's a part of.   At the end of the day when you're dealing with a large number of elements scattered across multiple files, it just makes things a lot more maintainable -- never force future engineers to change something in multiple places if it's possible to let them change it in just one place.